src/commands/notify.yml

```diff
@@ -67,11 +67,12 @@ parameters:
     type: boolean
     default: true
 
-  dashboard_link:
-    description: The URL to the dashboard
+  lighthouse_button_links:
```

Comment:
We are making this generic. Please remove the references to lighthouse

Reply:
done